README.md

```diff
@@ -151,3 +151,39 @@ To test the NGINX Kubernetes Gateway run:
 ```
 make unit-test
 ```
+
+# Release Process for NGINX Kubernetes Gateway
+
+### NGINX Kubernetes Gateway will use semantic versioning for releases described in semver.org.. 
+
+Given a version number MAJOR.MINOR.PATCH, increment the:
+
+- MAJOR version when you make incompatible API changes
+- MINOR version when you add functionality in a backwards compatible manner
+- PATCH version when you make backwards compatible bug fixes
+
+Note: While this project is using a beta version, it's important to point to this rule from https://semver.org/#spec-item-4.
+"Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable."
+
+
+### Steps to create a release.
+
+1. Create a release branch in the format release-X.Y from the main branch.
+
+2. Create a release candidate tag in the format vX.Y.Z-rc.n starting with n set to 1.
+
+3. Test the release candidate.
+
+    If the tests fail
+
+    - Create a fix for the error.
+
+    - Open a PR with the fix against the main branch.
+
+    - Once approved and merged, cherry-pick the commit into the release branch.
+
+    - Create a new release candidate tag from the release branch with n incremented in the tag name, vX.Y.Z-rc.n.
```

Comment:
Create a new release candidate tag, increment the release candidate number by 1.

Reply:
Thanks!  I made this update.